JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts

```diff
@@ -156,6 +166,23 @@ module Microsoft.ApplicationInsights.Telemetry.Common {
             return measurements;
         }
 
+        public static sanitizeId(id: string) {
+            if (id) {
+                id = Util.trim(id);
+                if (id.length > DataSanitizer.MAX_ID_LENGTH) {
+                    id = id.substring(0, DataSanitizer.MAX_ID_LENGTH);
+                    _InternalLogging.throwInternal(
+                        LoggingSeverity.WARNING,
+                        _InternalMessageId.UrlTooLong,
+                        "id is too long, it has been truncated to " + DataSanitizer.MAX_URL_LENGTH + " characters.",
```

Comment:
the message has a different value (url length) than the checks did (id length)

Reply:
(and has messageid url too long)

Reply:
Fixed.